wavenet.py

```diff
@@ -171,7 +188,7 @@ def _create_network(self, input_batch):
 
             # We skip connections from the outputs of each layer, adding them
             # all up here.
-            total = sum(outputs)
+            total = tf.add_n(outputs)
```

Comment:
I thought tensorflow parallelizes all its ops for GPU. I was worried that the built-in python sum would end up creating tensors for each intermediate summation result. That also seems slower because, depending on how it gets implemented, has to write the intermediate tensor results to global memory multiple times.

But it really is an unrelated change to the intent of this PR so I guess it should be done separately if at all.